client/app/states/templates/editor/editor.html

```diff
@@ -0,0 +1,9 @@
+<ol class="breadcrumb">
```

Comment:
Not the way we do breadcrumbs anymore, check any of the explorer component headers for the pattern.